File: playground/src/main/java/com/jiamian/translation/entity/dto/ImagesApiDTO.java
package com.jiamian.translation.entity.dto;

import com.google.common.collect.Lists;
import lombok.Data;

import java.util.List;

/**
 * @author DingGuangHui
 * @date 2023/3/28
 */
@Data
public class ImagesApiDTO {

	private String url = "";
	private boolean nsfw = false;
	private int width;
	private int height;
	private String hash = "";
	private int userId;
	private String generationProcess = "";
	private boolean needsReview = false;
	private String scannedAt = "";
	private List<String> tags = Lists.newArrayList();
	private MetaApiDTO meta;
}

File: web/src/main/java/com/jiamian/translation/entity/dto/api/ImagesApiDTO.kt
package com.jiamian.translation.entity.dto.api

import com.google.common.collect.Lists
import lombok.Data

/**
 * @author DingGuangHui
 * @date 2023/3/28
 */
@Data
class ImagesApiDTO {
    var url: String? = ""
    var nsfw: Boolean? = false
    var width = 0
    var height = 0
    var hash: String? = ""
    var userId: Long? = 0
    var generationProcess: String? = ""
    var needsReview: Boolean? = false
    var scannedAt: String? = ""
    var tags: List<String> = Lists.newArrayList()
    var meta: MetaApiDTO? = null
}